docs/docs/products/parachain-vault-strategy.md

```diff
@@ -1,4 +1,4 @@
-# Parachain Vault Strategy (discontinued)
```

Comment:
(discontinued)?

Reply:
This document was moved from the original book/ as-is. discontinued was already there